migrate.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+GOROOT=${1:-../go}
```

Comment:
`go env GOOOT` as a fallback?

Reply:
That would use your system GOROOT, which I'm not sure is the right choice either.